stan/math/rev/functor/cvodes_integrator.hpp

```diff
@@ -183,8 +183,7 @@ class cvodes_integrator {
    * @throw <code>std::invalid_argument</code> if arguments are the wrong
    *   size or tolerances or max_num_steps are out of range.
    */
-  cvodes_integrator(const char* function_name, const F& f,
-                    const Eigen::Matrix<T_y0, Eigen::Dynamic, 1>& y0,
+  cvodes_integrator(const char* function_name, const F& f, const T_y0& y0,
```

Comment:
[optional] Should there be a require on `T_y0`?

Reply:
fixed